framework/wazuh/core/utils.py

```diff
@@ -1884,16 +1884,21 @@ def validate_wazuh_xml(content: str, config_file: bool = False):
     WazuhError(1113)
         XML syntax error.
     """
+    # Well-formed XML documents should start with a '<' (excluding whitespaces)
+    # https://www.w3.org/TR/REC-xml/#dt-wellformed
+    if not content.lstrip().startswith('<'):
+            raise WazuhError(1113)
```

Comment:
This is not strictly necessary (perhaps redundant) since the validation would fail below when doing the parsing anyway. But given all XML files should start with a '<' character, we could save us some processing and fail early.